modules/update_data_table.nf

```diff
@@ -19,13 +21,17 @@ process update_data_table {
    if (production) {
       """
       cd /root/scripts/
-      python update_data_table.py -p /root/scripts/config.json -m "$comment" primary
+      python update_data_table.py -p /root/scripts/config.json -c $cohort -m "$comment" primary -pd
       """
-   }
-   else {
+   } else if (dry_run){
```

Comment:
I was wondering about this for my own ticket - could be a good time for discussion. 

Should we just set a standard for nextflow runs that we don't want to do `dry-run`s for when production mode is on? So `dry-run`s for all future nextflow BPC steps that we modify and add staging for, we only want to have it run for non-production mode. 

But if you really wanted to run `dry-run` for production mode, you technically could locally with the script. 

Thoughts on the above?

Reply:
Ya, `dry-run` might be useful when staging projects are not used. I would say having it in nextflow runs is not absolutely necessary for now. I vote for removing it.

Reply:
Since @Chelsea-Na is usually the run running nextflow runs, I'm wondering if you use `dry-run` for any BPC pipeline runs? If so, since we're adding staging project in so you can toggle between that and production on nextflow, do you still see a use case for using `dry-run` (nothing gets uploaded)?

Reply:
Just my 2 cents, sometimes we may want to be able to look at files locally, but we _could_ have that AND upload to staging.  May want to ping Chelsea in a DM, as not sure if her github notifications are working as expected

Reply:
@rxu17 I agree with the above - if we add staging, I don’t see a need for the `dry-run` option anymore. The main reason I was downloading the output locally was to verify it. Uploading it to a staging project would serve the same purpose.

Reply:
Thanks guys. I will remove the dry-run section.

Reply:
To clarify, we are just removing the dry-run for this nextflow module (not the script itself). We're still keeping dry-run as nextflow parameter because other BPC steps still use it.